frontend/ui/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -139,17 +150,16 @@ const MessageInput = (props: MessageInputProps & ConnectedProps<typeof connector
       const jsonTemplate = template.content;
 
       if (
-        jsonTemplate.message.text &&
-        !jsonTemplate.message.suggestions &&
-        !jsonTemplate.message.quick_replies &&
-        !jsonTemplate.message.containsRichText &&
-        !jsonTemplate.message.attachments &&
-        !jsonTemplate.message.attachment
+        jsonTemplate.text &&
```

Comment:
I think that at some point we will need helper functions like `isTextMessage` or `isRichCardMessage`

Reply:
I'd argue we're already at that point :D